src/scheduler/node_info.c

```diff
@@ -208,8 +209,10 @@ query_nodes(int pbs_sd, server_info *sinfo)
 	}
 #endif /* localmod 049 */
 
+	my_partitions = break_comma_list(partitions);
```

Comment:
break_comma_list() is allocating memory, you might want to check for NULL

Reply:
Done